src/routes/Issues/components/Issues.tsx

```diff
@@ -127,7 +127,6 @@ export default class Issues extends React.PureComponent<
 
     return (
       <div className="row issues-container">
-        <FixMeMetas title="Fixme | Issues" description="Find open issues of projects on the FixMe platform according to skill level and technologies of choice." />
```

Comment:
You are missing the description from here

Reply:
I think there is some bug with the FixMeMeta (or maybe Helmet) where it is recursively calling update and filling the call stack

Reply:
You are right, though you aren't setting the description as the meta tag with your current solution

Reply:
Is there any way to do it via vanilla js